utils/rowgroup/rowstorage.h

```diff
@@ -230,6 +236,13 @@ class RowAggStorage
     info = fCurData->fInfo[idx];
   }
 
+  void nextHashMultiplier()
+  {
+      // adding an *even* number, so that the multiplier will always stay odd. This is necessary
+      // so that the hash stays a mixing function (and thus doesn't have any information loss).
+      fCurData->hashMultiplier_ += 0xc4ceb9fe1a85ec54;
```

Comment:
Is this some theorem about using this constant, can you please clarify from did you take it?

Reply:
The constants for hash map are usually picked up and not calculated. It is borrowed from robin hood hashmap impl [here](https://github.com/martinus/robin-hood-hashing/blob/master/src/include/robin_hood.h#L2494).